chopper/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 4.0.0-nullsafety.0
+
+- Sound null safety support
```

Comment:
My bad, we should just go for "Null safety support" here if we're doing 4.0.0-nullsafety.0.
```suggestion
- Null safety support
```